libraries/Microsoft.Bot.Builder/CloudAdapterBase.cs

```diff
@@ -233,6 +233,13 @@ public override async Task CreateConversationAsync(string botAppId, string chann
             }
         }
 
+        /// <summary>
+        /// Gets streaming connector factory.
+        /// </summary>
+        /// <param name="activity">The activity.</param>
+        /// <returns>Streaming Connector Factory.</returns>
+        protected abstract ConnectorFactory GetStreamingConnectorFactory(Activity activity);
```

Comment:
I think some remarks are warranted here; it's a `protected` method that originated from a specific use case (proactive messaging + streaming channels). To make future maintainers and consumers of the SDK's lives easier, could you add additional context, @msomanathan?

Reply:
Sure, I'll add some context explaining the need for this method.